quartz/plugins/transformers/badges.ts

```diff
@@ -0,0 +1,241 @@
+import { QuartzTransformerPlugin } from "../types"
+import badgesCSS from "../../styles/badges.scss"
+// import badgesFixCSS from "../../styles/badges.fix.css"
+import icons from "lucide-static";
+
+export const BADGE_TYPES:any[] = [
```

Comment:
Is there a particular reason for the chosen icons? (Beside the ones we use for callouts)